bootstrap/syncmanager.go

```diff
@@ -240,12 +245,30 @@ func (manager *SyncManager) syncBlockWorker(blockHeight int64) ([]command_entity
 		return nil, fmt.Errorf("error requesting chain block_results at height %d: %v", blockHeight, err)
 	}
 
-	txs := make([]model.Tx, 0)
+	txs := make([]model.CosmosTxWithHash, 0)
 	for _, txHex := range block.Txs {
-		var tx *model.Tx
-		tx, err = manager.cosmosClient.Tx(parser.TxHash(txHex))
+		txHash := parser.TxHash(txHex)
+		tx := &model.CosmosTxWithHash{
+			Hash: txHash,
+		}
+
+		var resTx *model.Tx
+		resTx, err = manager.cosmosClient.Tx(txHash)
 		if err != nil {
-			return nil, fmt.Errorf("error requesting chain txs (%s) at height %d: %v", txHex, blockHeight, err)
+			if manager.TxDecoder != nil {
+				var decodedTx *model.CosmosTx
+				decoder := *manager.TxDecoder
+				decodedTx, err = decoder.DecodeBase64(txHex)
```

Comment:
```suggestion
				decodedTx, err = manager.TxDecoder.DecodeBase64(txHex)
```
I think this can become this once we fix the pointer to interface in the struct.